docs/local-deploy-dotnet-debugging-guide.md

```diff
@@ -0,0 +1,559 @@
+# Bicep Extension Debugging Guide
```

Comment:
Could we move this under `docs/experimental`, and link to it from https://github.com/Azure/bicep/blob/main/docs/experimental/local-deploy.md?